openstack_exporter/collectors/cinderbackend.py

```diff
@@ -0,0 +1,131 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+from prometheus_client.core import GaugeMetricFamily
+
+from openstack_exporter import BaseCollector
+
+LOG = logging.getLogger('openstack_exporter.exporter')
+
+
+class CinderBackendCollector(BaseCollector.BaseCollector):
+
+    def describe(self):
+        yield GaugeMetricFamily('cinder_total_capacity',
+                                'Cinder total capacity in GiB')
+        yield GaugeMetricFamily('cinder_free_capacity',
+                                'Cinder free capacity in GiB')
+        yield GaugeMetricFamily('cinder_allocated_capacity',
+                                'Cinder allocated capacity in GiB')
```

Comment:
Would rename metrics to have unit in metric name straight:
`cinter_total_capacity_gib` + rest

Reply:
added